### DISCUSSION TITLE
可视化编辑器是否支持linker field和caculator field

### DISCUSSION THREAD

**Discussion by chenshuichao** [alibaba/formily#2012] [Q&A 2.x] [answered]
form builder挺好的，谢谢！
我想确认的是，可视化编辑器，是否支持关联，计算，这两种字段的编辑和设计。
如果目前不支持的话，是否有时间表。
再次感谢！非常棒的工作！

**Comment by janryWang** [comment, marked as answer]
响应器配置，能解决你的需求，多研究研究吧

**Reply by chenshuichao** [reply]
好的，谢谢！
就是不知道，傻瓜不傻瓜，一般的报表设计器，带的公式编辑器，还是挺傻瓜的。

**Reply by janryWang** [reply]
这个是个开源项目，能给大家用就不错了，要求还这么高？还有，本身dn的扩展性就好，随便你定制
